ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -564,6 +564,37 @@ tests:
           cpu: 100m
           memory: 200Mi
       timeout: 6h0m0s
+- as: osde2e-stage-aws-addon-mcg-osd
+  cron: 0 6,18 * * *
+  steps:
+    test:
+    - as: addon-mcg-osd-tests
+      commands: |
+        export ADDON_IDS="mcg-osd"
```

Comment:
@rexagod It's required to add [ADDON_PARAMETERS](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml#L530) as there is 1 required parameter for the addon.

Reply:
ACK